frame/staking/Cargo.toml

```diff
@@ -20,6 +20,11 @@ pallet-session = { version = "2.0.0", features = ["historical"], path = "../sess
 pallet-authorship = { version = "2.0.0", default-features = false, path = "../authorship" }
 sp-application-crypto = { version = "2.0.0", default-features = false, path = "../../primitives/application-crypto" }
 
+# Needed for benchmarking stuff. TODO: this stuff must NOT be always compiled. use a feature.
+pallet-indices = { version = "2.0.0", default-features = false, path = "../indices" }
```

Comment:
cc @DarkEld3r

Reply:
Could we not put the benchmarking into its own crate?

Reply:
We sure could. Question is, is it needed given 
https://github.com/paritytech/substrate/issues/4873

Reply:
As you know, features are broken in Rust/Cargo. I like to prevent their usage as much as possible. But yeah, as long as you don't have diverging code bases based on the feature, "it should be okay"